LambdaSource/template.yaml

```diff
@@ -74,6 +84,9 @@ Resources:
           HookUrl: !Ref HookUrl
           Region: !Ref 'AWS::Region'
           api_token: !Ref apiToken
+          blacklist: "Not set yet"
+          GroupName: "Team-Bits,Team-SRE,Team-DonkeyKong,Team-Mercury,Team-Outbounders,Team-TopGun,Team-Voyager"
```

Comment:
Remove team names and slack channels - create new commit and force push the branch to overwrite history